themes/digital.gov/layouts/partials/core/community-join-without-form.html

```diff
@@ -1,20 +1,25 @@
 {{/* Listserv Platforms without form sign-up option */}}
 
-<a href="mailto:{{- .e.subscribe_email -}}
-{{- if .e.subscribe_email_subject -}}
-  ?subject={{- .e.subscribe_email_subject -}}
-  {{- if .e.subscribe_email_body -}}
-    &body={{- .e.subscribe_email_body -}}
-  {{- end -}}
-{{- else -}}
-  {{- if .e.subscribe_email_body -}}
-    ?body={{- .e.subscribe_email_body -}}
-  {{- end -}}
-{{- end -}}" class="join type-{{- .e.platform -}}">
-  <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
-    <use xlink:href="{{ "uswds/img/sprite.svg#mail" | relURL }}"></use>
-  </svg>
-  Join by email
-</a>
+<p>Email <strong>{{ .e.subscribe_email }}</strong> to join. {{ if .e.subscribe_email_subject }} Include <strong>{{ .e.subscribe_email_subject }}</strong> in the subject line{{ else }} Leave the subject line empty{{ end }}{{- if .e.subscribe_email_body }} and <strong>{{ .e.subscribe_email_body }}</strong> in the body{{- end -}}.</p>
+
+<div class="join-button">
```

Comment:
Is there another previously established component class we can use that will give us this button style?

We should avoid adding custom classes and styles and favor current components.

Reply:
I have removed the `join-button` div with the new markup for style changes that have been made. Good point!